src/main/java/com/redhat/cloud/policies/app/auth/models/Access.java

```diff
@@ -18,9 +18,48 @@
 
 import java.util.List;
 
-public class Access {
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public class Access extends RbacRawCommon {
```

Comment:
As I understand, `RbacRawCommon` only expose some constants. You don't need to inherit from it.

Reply:
I had a common methods there at first. But, yeah it can be a static class with constants.

Reply:
I'll keep it as is.